test/functional/services/test_subjects.ts

```diff
@@ -169,10 +169,14 @@ export function TestSubjectsProvider({ getService }: FtrProviderContext) {
       });
     }
 
```

Comment:
Underlying selenium method allows for a timeout, so I added it as an optional param here.